server/routing/lobby.go

```diff
@@ -137,15 +164,18 @@ func (lr *LobbyRoutes) lobbyStartGameHandler(w http.ResponseWriter, r *http.Requ
 		GameState: gameInstance.State,
 	}
 
-	// tell all pubsubbers
+	w.Write([]byte("The game has started."))
+
+	PublishMessage(msg, lobby)
+	return
+}
+
+func PublishMessage(msg pogo.Typer, lobby services.Lobby) {
```

Comment:
I like this encapsulation a lot.  Thank you for doing that.  I knew we needed to, but I wasn't exactly sure how to structure it.